libraries/compute-utils/function-views/src/variance-based-analysis/sa-outputs-routine.ts

```diff
@@ -136,3 +136,33 @@ export function getOutput(funcCalls: DG.FuncCall[], outputsSpecification: Output
 
   return table;
 }
+
+export function getInputOutputColumns(inputs: DG.Column[], outputs: DG.Column[]): DG.Column[] {
+  outputs.forEach((outCol) => {
+    inputs.forEach((inCol) => {
+      if (inCol.name === outCol.name) {        
+        inCol.name = `${inCol.name} (input)`;
+        outCol.name = `${outCol.name} (output)`;
+      }
+    })
+  });
+
+  return inputs.concat(outputs);
+};
+
+export function getDataFrameFromInputsOutputs(inputs: DG.Column[], outputs: DG.Column[]): DG.DataFrame {
```

Comment:
Isn't it better to create input/output columns with proper names initially (before calling `getDataFrameFromInputsOutputs`) ? 
This method mutates input parameters, which is no good for understanding and debugging

Reply:
Agree. But, actually, I don't know how to implement this in the most compact way. This routine is used in both Random & Sobol methods.

My proposition is: to modify function name and add comments

Reply:
Proper solution requires efforts and time to be done.
There is no need in creating `DG.Column` objects till the resulting DF is created. They could be created in the very end of the `perform` method. Until then, you may use simple structures and constuct the DF and columns only once before actually returning the analysis result.

This all could be done in refactoring stage. If something urgent should be delivered, I am OK with this PR

Reply:
Thanks